docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -110,9 +110,84 @@ spec:
     - url: "github.com/kyma-project/addons//addons/index.yaml"
     # bitbucket.org URL with no prefix. It is automatically interpreted as a Git repository source.
     - url: "bitbucket.org/kyma-project/addons//addons/index.yaml"
+    # Git SSH protocol with a reference to a secret containing base64 encoded SSH private key
```

Comment:
u added that entry here but not for HTTP(s), let's make it symmetric. Add it in both places or remove it from both places